tests/crashes/133199.rs

```diff
@@ -0,0 +1,11 @@
+//@ known-bug: #133199
+//@ aux-build: aux133199.rs
```

Comment:
Discussion: do we want this as a crashes test given we know GCE is incomplete cc @BoxyUwU

Reply:
> But yes, every issue should ideally have a test. ICE issues are not special here.